### ISSUE TITLE
Plugin weekOfYear doesn't work well for locale zh-cn

### ISSUE THREAD

**Issue by DevinXian** [iamkun/dayjs#664]
**Issue code demo**
```
const dayjs = require('dayjs')
const moment = require( 'moment')
require( 'dayjs/locale/zh-cn')
const weekOfYear = require('dayjs/plugin/weekOfYear')

moment.locale('zh-cn')

dayjs.locale('zh-cn')
dayjs.extend(weekOfYear)

console.log(dayjs('2019-08-18').week()) // 34
console.log(moment('2019-08-18').week()) // 33
```
**Expected behavior**

Both should be 33 

**Information**
 - Day.js Version [v1.8.15]
 - Moment Version [v2.24.0]
 - OS: [Mac OS X 10.14.4]
 - Time zone: [GMT+08:00(China Standard Time)]

**Comment by PengJiyuan** [comment]
+1

**Comment by iamkun** [comment]
fixed in #658
